miniconda3-nanoserver/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM mcr.microsoft.com/windows/servercore:1809 AS servercorebase
+
+WORKDIR /tmp
+
+RUN powershell (New-Object System.Net.WebClient).DownloadFile('https://repo.continuum.io/miniconda/Miniconda3-latest-Windows-x86_64.exe', 'Miniconda3.exe')
```

Comment:
What was the thought for splitting these up into their own layers opposed to just one big layer to optimize image size?

Reply:
This is just the build/download stage.... this is only important for the final stage